doc/res-links.md

```diff
@@ -0,0 +1,25 @@
+<!---
+ Copyright (c) 2016 StrongLoop, IBM, and Express Contributors
+ License: MIT
+-->
+
+<h3 id='res.links'>res.links(links)</h3>
+
+Joins the `links` provided as properties of the parameter to populate the response's
+`Link` HTTP header field.
+
+For example, the following call:
+
+~~~js
+res.links({
+  next: 'http://api.example.com/users?page=2',
+  last: 'http://api.example.com/users?page=5'
+});
+~~~
+
+Yields the following results:
+
+~~~js
```

Comment:
True, of course, but the syntax highlighting works fine "as is" so I don't think we need to change it.   If we do, then we need to account for it in the "sync script" in https://github.com/strongloop/expressjs.com/pull/582.